spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -98,70 +98,70 @@ Use of implementations beyond Jakarta Annotations 1.3 in MicroProfile is allowed
  - https://github.com/eclipse-ee4j/common-annotations-api
 
 [[mp-config]]
-=== MicroProfile Config 2.0
+=== MicroProfile Config 3.0-RC4
```

Comment:
Before merging the PR, please look up if the specs have not released any now RC's yet

Reply:
Please use 3.0-RC5